ThunderCloud/AccordionTabBarViewController.swift

```diff
@@ -55,24 +55,26 @@ public class AccordionTabBarItem: Row {
     
     let viewController: UIViewController
     
-    public init(viewController: UIViewController) {
-        
+    let navigationBarHeight: CGFloat
+    
+    public init(viewController: UIViewController, navigationBarHeight: CGFloat) {
+        self.navigationBarHeight = navigationBarHeight
         self.viewController = viewController
     }
     
     public var remainingHeight: CGFloat = 0
     
     private var viewControllerHeight: CGFloat {
-        return searchBar != nil ? remainingHeight - 56 : (titleView != nil ? remainingHeight - 44 : remainingHeight)
+        return searchBar != nil ? remainingHeight - navigationBarHeight : (titleView != nil ? remainingHeight - navigationBarHeight : remainingHeight)
     }
     
     public var expanded: Bool = false
     
     public var isFirstItem: Bool = false
     
     public func height(constrainedTo size: CGSize, in tableView: UITableView) -> CGFloat? {
-        let height = expanded ? remainingHeight + 44 : 44
-        return isFirstItem ? height + 20 : height
+        let height = expanded ? remainingHeight + navigationBarHeight : navigationBarHeight
+        return isFirstItem ? height + UIApplication.shared.statusBarFrame.height : height
```

Comment:
I know the `statusBarFrame` is fetched different when using iOS 13 `scenes` but guess this is OK if we don't use that?

Reply:
Yeah don't think we'd have to worry too much about that for now :) A lot of Storm will need a re-think once we start using SwiftUI anyways :D